eslint/javascript/eslint-config-clever-cloud-esm.js

```diff
@@ -0,0 +1,49 @@
+import js from '@eslint/js';
+import importPlugin from 'eslint-plugin-import';
+import globals from 'globals';
+
+export default {
+  name: 'eslint-config-clever-cloud-esm',
+  files: ['**/*.js', '**/*.mjs'],
+  linterOptions: {
+    reportUnusedDisableDirectives: 'warn',
+  },
+  languageOptions: {
+    globals: {
+      ...globals.browser,
+    },
+    ecmaVersion: 2022,
+    sourceType: 'module',
+    parserOptions: {
+      ecmaFeatures: {
+        impliedStrict: true,
+      },
+    },
+  },
+  plugins: {
+    import: importPlugin,
+  },
+  rules: {
+    ...js.configs.recommended.rules,
+    ...importPlugin.flatConfigs.recommended.rules,
+    'accessor-pairs': 'off',
+    eqeqeq: ['error', 'always', { null: 'never' }],
+    camelcase: ['error', { allow: ['_lp$'] }],
+    curly: ['error', 'all'],
+    'no-new': 'off',
+    'no-unused-vars': ['error', { argsIgnorePattern: '^_', varsIgnorePattern: '^_', ignoreRestSiblings: true }],
+    // import plugin
+    // some rules are disabled because TypeScript already handles it
```

Comment:
They don't but they are a lot less reliable, for instance the `import/named` cannot find some of our dependencies that TypeScript does.
I suspect it has something to do with wrong module resolution (CJS vs ESM as usual) but I couldn't find a good solution so after searching for a few hours to solve issues, I figured we were better off disabling the rules.

But you are absolutely right, if the project doesn't rely on TypeScript these rules could be good, and this config is supposed to become generic so it's a great feedback.

I'm not sure what to do at the moment, I'd like to check the fork of the import plugin to see if it handles these cases better. Maybe this can be done later on in another PR to avoid blocking this one, WDYT?

Reply:
Tracked in #1273, thanks @Galimede!